### PR TITLE
feature/KOJO-106 | infinite worker for SIGKILL and SIGTERM investigation

### PR DESCRIPTION
**Pull request by alexberryman**
- UseCase47 infinite worker for SIGKILL and SIGTERM investigation

### CODE REVIEW

File: UseCase47/src/V1/Worker/Delegate.php [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                'job_id' => $this->getApiV1WorkerService()->getJobId(),
+                'iterations' => $iterations,
+                'nested_object' => [
+                    'random_letters' => $this->randomLetterString(),
+                    'random_word' => $this->randomWord(),
+                    'random_value' => random_int(1, 100),
+                    'status' => 'success',
+                ],
+            ];
+            $this->getApiV1WorkerService()->getLogger()->notice('I can do this forever', $context);
```
Comment (mucha55) [inline UseCase47/src/V1/Worker/Delegate.php]: Heh

File: UseCase47/src/V1/Worker/Delegate.php [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+            $iterations++;
+            sleep(3);
```
Comment (mucha55) [inline UseCase47/src/V1/Worker/Delegate.php]: Oh man, if you were sleeping this long between iterations then there's no way you were hitting the default LA limit of 20

File: UseCase47/src/V1/Worker/Delegate.php (hunk 1 above)
Comment (rhift) [reply UseCase47/src/V1/Worker/Delegate.php]: ^

File: UseCase47/docker/build.sh [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+if [ "$COMPOSER_INSTALL" = true ]; then
+    echo "Installing composer dependencies..."
+    composer install -a
+fi
+
+if [ "$INSTALL_XDEBUG" = true ]; then
+    echo "Installing the xdebug extension..."
+    # An xdebug version grater than xdebug-2.7.0alpha1 is required due to https://bugs.xdebug.org/bug_view_page.php?bug_id=00000938
+    pecl install xdebug-2.7.0beta1
```
Comment (rhift) [inline UseCase47/docker/build.sh]:
```suggestion
    pecl install xdebug-2.7.2
```

**Pull request merged**